### PR TITLE
Minor fixes in Post details, Posts list in Admin page.

### PR DESCRIPTION
**Pull request by srikar0896**
### Refactor
Refactor to let the `fetchPosts` action to accept the JSON object to improve code readability

### Fixes
- Fixes - Highlight the current post on initial page load
- Fixes - Adding, delete comments and votes to a post does not get updated in the same post in the posts list.
- Fixes - Changing the status of a post in the post details does not persist with the applied filter in the posts list.
- Fixes - Updating the title of a post does not highlight the post in posts list(as we are depending on the post slug to highlight the selected post).
- Fixes - Though creating a post apply the filter `recent_activity` the `sort by` dropdown text still shows the previously applied filter.
- Fixes - Updating the post doesn't persist the applied filter in posts list.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def index
                      .reverse_chronologically
     end
 
-    @posts = paginate @posts, per_page: 25
+    @posts = paginate @posts, per_page: 7
```
Comment (srikar0896) [inline app/controllers/posts_controller.rb]:
```suggestion
    @posts = paginate @posts, per_page: 25
```

**Comment by pranav7** [comment]
There is definitely something weird going on here 🙂 

2020-04-03 at 5 09 PM

File: app/javascript/common/eventBus.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+class EventBus {
```
Comment (pranav7) [inline app/javascript/common/eventBus.js]: What's the idea behind the `EventBus`?

Comment (pranav7) [reply app/javascript/common/eventBus.js]: We should also follow one naming convention for file names, I think we have all others as upper camel case?

**Pull request merged**